server/settings/main.py

```diff
@@ -276,31 +277,38 @@ class CoreSettings(BaseSettingsModel):
     disk_mapping: DiskMappingModel = SettingsField(
         default_factory=DiskMappingModel,
         title="Disk mapping",
+        description="Map virtual disk(s) on artist's machine for AYON when running",
     )
     tools: GlobalToolsModel = SettingsField(
         default_factory=GlobalToolsModel,
-        title="Tools"
+        title="Tools",
+        description="Configure basic aspects for workfiles, products, and publishing",
     )
     version_start_category: VersionStartCategoryModel = SettingsField(
         default_factory=VersionStartCategoryModel,
-        title="Version start"
+        title="Version start",
+        description="Configure versioning for products using profiles",
     )
     imageio: CoreImageIOBaseModel = SettingsField(
         default_factory=CoreImageIOBaseModel,
-        title="Color Management (ImageIO)"
+        title="Color Management (ImageIO)",
+        description="Enable global color management and configure OCIO",
     )
     publish: PublishPuginsModel = SettingsField(
         default_factory=PublishPuginsModel,
-        title="Publish plugins"
+        title="Publish plugins",
+        description="Configure various aspects of publishing process and components",
```

Comment:
Way too generic? Or? Does such a description really help?

Reply:
Again, I get your point, but how you approach a title (and short line of text) when the category has so many items...it must be sort of generic. To me, it tells user what he can expect there - Publish plugins just tells : some list of addons present, but it actually tells nothing about the publishing itself can be tweaked too...works for me (but could be better formated probably as anything else imho)

Reply:
I can think of e.g. such description instead: ```Configure the way publishing performs and components it produces```

?

Reply:
> Again, I get your point, but how you approach a title (and short line of text) when the category has so many items...it must be sort of generic. To me, it tells user what he can expect there - Publish plugins just tells : some list of addons present, but it actually tells nothing about the publishing itself can be tweaked too...works for me (but could be better formated probably as anything else imho)

Id'd argue if it's too generic in the description - it likely means we should focus on adding description of the stuff inside of it instead of just describing "there's publish plugins stuff here" on a "Publish Plugins" section :D

Reply:
will change it accordingly